feedservice/redis/redis.go

```diff
@@ -0,0 +1,70 @@
+package redis
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gomodule/redigo/redis"
+)
+
+const (
+	NVT_FILENAME_POS           = iota //0
```

Comment:
you can assign them manually then you don't have to put a comment to identify the number :) also it's is more explicit than ioata.